mmv1/third_party/terraform/services/servicedirectory/resource_service_directory_endpoint_test.go.tmpl

```diff
@@ -96,3 +96,103 @@ resource "google_service_directory_endpoint" "example" {
 }
 `, testId, location, testId, testId)
 }
+
+{{ if ne $.TargetVersionName "ga" }}
```

Comment:
Why do we need beta-only tests for this?

Reply:
this would actually only be necessary when we do have a beta field only. RIght now the only difference is metadata/annotations change which has been resolved. removing this in latest commit.